diff --git a/bin/mx-scripts.js b/bin/mx-scripts.js
index 386ff15..f6f8c0f 100755
--- a/bin/mx-scripts.js
+++ b/bin/mx-scripts.js
@@ -112,7 +112,7 @@ function getRealCommand(cmd, toolsRoot) {
 }
 
 function findNodeModulesBin() {
-    let parentDir = join(__dirname, "..");
+    let parentDir = join(__dirname, "../..");
     const bins = [];
     while (parse(parentDir).root !== parentDir) {
         const candidate = join(parentDir, "node_modules/.bin");
diff --git a/configs/eslint.ts.base.json b/configs/eslint.ts.base.json
index 660df6f..20ae95b 100644
--- a/configs/eslint.ts.base.json
+++ b/configs/eslint.ts.base.json
@@ -18,7 +18,8 @@
         "plugin:@typescript-eslint/recommended",
         "prettier",
         "plugin:prettier/recommended",
-        "plugin:react-hooks/recommended"
+        "plugin:react-hooks/recommended",
+        "plugin:react/jsx-runtime"
     ],
     "parser": "@typescript-eslint/parser",
     "parserOptions": {
diff --git a/configs/rollup-plugin-collect-dependencies.mjs b/configs/rollup-plugin-collect-dependencies.mjs
index 2d0b5bf..92291d4 100644
--- a/configs/rollup-plugin-collect-dependencies.mjs
+++ b/configs/rollup-plugin-collect-dependencies.mjs
@@ -2,9 +2,8 @@
 
 import fg from "fast-glob";
 import fsExtra from "fs-extra";
-import { existsSync, readFileSync, writeFileSync } from "fs";
+import { cpSync, existsSync, lstatSync, readFileSync, realpathSync, writeFileSync } from "fs";
 import { dirname, join, parse } from "path";
-import copy from "recursive-copy";
 import { promisify } from "util";
 import resolve from "resolve";
 import _ from "lodash";
@@ -171,14 +170,28 @@ async function copyJsModule(moduleSourcePath, to) {
     if (existsSync(to)) {
         return;
     }
-    return promisify(copy)(moduleSourcePath, to, {
-        filter: [
-            "**/*.*",
-            LICENSE_GLOB,
-            "!**/{android,ios,windows,mac,jest,github,gradle,__*__,docs,jest,example*}/**/*",
-            "!**/*.{config,setup}.*",
-            "!**/*.{podspec,flow}"
-        ]
+
+    let actualSourcePath = moduleSourcePath;
+    if (lstatSync(moduleSourcePath).isSymbolicLink()) {
+        actualSourcePath = realpathSync(moduleSourcePath);
+    }
+
+    cpSync(actualSourcePath, to, {
+        recursive: true,
+        dereference: true,
+        filter: src => {
+            const relativePath = src.replace(actualSourcePath, "").replace(/^[\\/]/, "");
+
+            if (relativePath.match(/[\\/](android|ios|windows|mac|jest|github|gradle|__.*__|docs|example.*)[\\/]/)) {
+                return false;
+            }
+
+            if (relativePath.match(/\.(config|setup)\.|\.podspec$|\.flow$/)) {
+                return false;
+            }
+
+            return true;
+        }
     });
 }
 
