diff --git a/configs/rollup.config.native.mjs b/configs/rollup.config.native.mjs
index 62c5c82ddbd628e5d22709660f26e82855ddfceb..927c8e729238f8cb2971b752c2ccace3d9464606 100644
--- a/configs/rollup.config.native.mjs
+++ b/configs/rollup.config.native.mjs
@@ -52,9 +52,9 @@ const nativeExternal = [
     /^react($|\/)/,
     /^react-native-gesture-handler($|\/)/,
     /^react-native-reanimated($|\/)/,
-    /^react-native-fast-image($|\/)/,
+    /^react-native-worklets($|\/)/,
     /^react-native-svg($|\/)/,
-    /^react-native-vector-icons($|\/)/,
+    /^@react-native-vector-icons\/common($|\/)/,
     /^@?react-navigation($|\/)/,
     /^react-native-safe-area-context($|\/)/,
     /^@d11\/react-native-fast-image($|\/)/
@@ -222,7 +222,7 @@ export default async args => {
                   })
                 : null,
             image(),
-            production ? terser({ mangle: false }) : null,
+            production ? terser({ mangle: false, compress: { directives: false } }) : null,
             // We need to create .mpk and copy results to test project after bundling is finished.
             // In case of a regular build is it is on `writeBundle` of the last config we define
             // (since rollup processes configs sequentially). But in watch mode rollup re-bundles only
diff --git a/test-config/jest.native.config.js b/test-config/jest.native.config.js
index 9424f7f802a4ca7843d536d4d46c67375e73f67f..af13d476c202d712cab51e5a83be9d2183fc1008 100644
--- a/test-config/jest.native.config.js
+++ b/test-config/jest.native.config.js
@@ -16,8 +16,9 @@ module.exports = {
         ...(hasDependency("react-native-gesture-handler") ? ["react-native-gesture-handler/jestSetup.js"] : [])
     ],
     testMatch: ["<rootDir>/**/*.spec.{js,jsx,ts,tsx}"],
-    transformIgnorePatterns: ["node_modules/(?!(.*react-native.*|victory-)/)"],
+    transformIgnorePatterns: ["node_modules/(?!(.*react-native.*|victory-|@shopify/flash-list)/)"],
     transform: {
+        "^.+\\.(ttf|otf|woff2?|eot)$": require.resolve("react-native/jest/assetFileTransformer.js"),
         "^.+\\.tsx?$": [
             "ts-jest",
             {
