diff --git a/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js b/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js
index ec507d5..f5bf176 100755
--- a/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js
@@ -1,6 +1,6 @@
 #! /usr/bin/env node
 const { execSync, spawnSync } = require("child_process");
-const { existsSync } = require("fs");
+const { existsSync, lstatSync } = require("fs");
 const { delimiter, dirname, join, parse } = require("path");
 const { checkMigration } = require("../utils/migration");
 const { red } = require("ansi-colors");
@@ -14,9 +14,10 @@ checkNodeVersion();
     }

     const [, currentScriptPath, cmd, ...args] = process.argv;
-    const toolsRoot = currentScriptPath.endsWith("pluggable-widgets-tools")
+    const isSymlink = lstatSync(currentScriptPath).isSymbolicLink();
+    const toolsRoot = currentScriptPath.endsWith("pluggable-widgets-tools") && !isSymlink
         ? join(dirname(currentScriptPath), "../@mendix/pluggable-widgets-tools")
-        : join(dirname(currentScriptPath), "..");
+        : isSymlink ? join(__dirname, "../") : join(dirname(currentScriptPath), "..");

     if (args.indexOf("--subprojectPath") > -1) {
         args.splice(args.indexOf("--subprojectPath"), 2);
diff --git a/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js b/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js
index 85165f8..aabe7af 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js
@@ -277,6 +277,6 @@ const nativeExternal = [
     /^react-native-fast-image($|\/)/,
     /^react-native-svg($|\/)/,
     /^react-native-vector-icons($|\/)/,
-    /^react-navigation($|\/)/,
+    /^@?react-navigation($|\/)/,
     /^react-native-safe-area-context($|\/)/
 ];
diff --git a/node_modules/@mendix/pluggable-widgets-tools/test-config/jest.native.config.js b/node_modules/@mendix/pluggable-widgets-tools/test-config/jest.native.config.js
index 7e604c2..62afe21 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/test-config/jest.native.config.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/test-config/jest.native.config.js
@@ -33,7 +33,6 @@ module.exports = {
         "^.+\\.tsx?$": [
             "ts-jest",
             {
-                isolatedModules: true,
                 tsconfig: { module: "commonjs" }
             }
         ],
diff --git a/node_modules/@mendix/pluggable-widgets-tools/test-config/test-index-native.js b/node_modules/@mendix/pluggable-widgets-tools/test-config/test-index-native.js
index ac385ec..63b3871 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/test-config/test-index-native.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/test-config/test-index-native.js
@@ -5,6 +5,8 @@ const enableHooks = require("jest-react-hooks-shallow").default;
 configureEnzyme({ adapter: new Adapter() });
 enableHooks(jest);

+global.setImmediate = global.setTimeout;
+
 const origConsole = console.error;
 const BLOCKED_ERROR_TAGS = [
     "prop on a DOM element",