diff --git a/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js b/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js
index 34e0877..510cec4 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/configs/rollup.config.native.js
@@ -63,7 +63,8 @@
     /^react-native-svg($|\/)/,
     /^react-native-vector-icons($|\/)/,
     /^@?react-navigation($|\/)/,
-    /^react-native-safe-area-context($|\/)/
+    /^react-native-safe-area-context($|\/)/,
+    /^react-native-view-shot($|\/)/
 ];
 
 export default async args => {
