diff --git a/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js b/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js
index 7fc60e0..667c35a 100755
--- a/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/bin/mx-scripts.js
@@ -1,12 +1,14 @@
 #! /usr/bin/env node
 const { spawnSync } = require("child_process");
-const { existsSync } = require("fs");
+const { existsSync, lstatSync } = require("fs");
 const { delimiter, dirname, join, parse } = require("path");

 const [, currentScriptPath, cmd, ...args] = process.argv;
-const toolsRoot = currentScriptPath.endsWith("pluggable-widgets-tools")
+const isSymlink = lstatSync(currentScriptPath).isSymbolicLink();
+
+const toolsRoot = currentScriptPath.endsWith("pluggable-widgets-tools") && !isSymlink
     ? join(dirname(currentScriptPath), "../@mendix/pluggable-widgets-tools")
-    : join(dirname(currentScriptPath), "..");
+    : isSymlink ? join(__dirname, "../") : join(dirname(currentScriptPath), "..");

 if (args.indexOf("--subprojectPath") > -1) {
     args.splice(args.indexOf("--subprojectPath"), 2);
diff --git a/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeIcon.js b/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeIcon.js
index f8b602b..aa47292 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeIcon.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeIcon.js
@@ -1,4 +1,6 @@
-import { createElement } from "react";
-import { View } from "react-native";
+const { createElement } = require("react");
+const { View } = require("react-native");

-export const Icon = () => createElement(View, { testId: "icon" });
+module.exports = {
+    Icon: () => createElement(View, { testId: "icon" })
+}
diff --git a/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeImage.js b/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeImage.js
index f787092..feebc48 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeImage.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/test-config/__mocks__/NativeImage.js
@@ -1 +1,3 @@
-export const Image = require("react-native").Image;
+module.exports = {
+    Image: require("react-native").Image
+};
