diff --git a/DefaultMarker.js b/DefaultMarker.js
index 618d916..bb30f07 100644
--- a/DefaultMarker.js
+++ b/DefaultMarker.js
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 import { View, StyleSheet, Platform, TouchableHighlight } from 'react-native';
 
-const ViewPropTypes = require('react-native').ViewPropTypes || View.propTypes;
+import { ViewPropTypes } from 'deprecated-react-native-prop-types'
 
 export default class DefaultMarker extends React.Component {
   static propTypes = {
diff --git a/MultiSlider.js b/MultiSlider.js
index 3a5e417..e48b97b 100755
--- a/MultiSlider.js
+++ b/MultiSlider.js
@@ -12,8 +12,7 @@ import {
 
 import DefaultMarker from './DefaultMarker';
 import { createArray, valueToPosition, positionToValue } from './converters';
-
-const ViewPropTypes = require('react-native').ViewPropTypes || View.propTypes;
+import { ViewPropTypes } from 'deprecated-react-native-prop-types'
 
 export default class MultiSlider extends React.Component {
   static propTypes = {
